docs/markdown/get_started/developers/development_process.md

```diff
@@ -166,14 +168,14 @@ When pushing new changes to GitHub, your PR title should be aligned with the sco
 
 Our versioning guidelines follow those outlined at **[semver.org](https://semver.org)**:
 
-- Patch: internal fixes, documentation changes, or package upgrades (anything that consumers of Gestalt don't need to worry about)
```

Comment:
Check the get_started/developers/development_process#Guidelines site, I updated the info re Codemods. Maybe try rebasing?

Reply:
Looks like the section is removed? I went ahead and tried to match them.

Reply:
All that info was removed because it moved to Releases section